lib/diagnostic.rb

```diff
@@ -35,8 +44,8 @@
 # "Somerville". Finally, assign the modified Person to `Response.dave` below.
 
 ## replace nil with your response, then continue your work on the next line
-dave = nil
-Response.dave = dave
+dave = Person.new("Dave", 32, "Ohio")
+:location.dave = Somerville
```

Comment:
This will attempt to invoke a method (or set a property) named `dave` on the symbol `:location` and set it equal to an identifier named `Somerville`.  It will cause an error.  `dave.location = 'Somerville'`.